libpod/adapter/containers_remote.go

```diff
@@ -0,0 +1,62 @@
+// +build remoteclient
+
+package adapter
+
+import (
+	"encoding/json"
+
+	iopodman "github.com/containers/libpod/cmd/podman/varlink"
+	"github.com/containers/libpod/libpod"
+	"github.com/containers/libpod/pkg/inspect"
+)
+
+// Inspectr returns an inspect struct from varlink
+func (c *Container) Inspect(size bool) (*inspect.ContainerInspectData, error) {
+	reply, err := iopodman.ContainerInspectData().Call(c.Runtime.Conn, c.ID())
+	if err != nil {
+		return nil, err
+	}
+	data := inspect.ContainerInspectData{}
+	if err := json.Unmarshal([]byte(reply), &data); err != nil {
+		return nil, err
+	}
+	return &data, err
+}
+
+// ID returns the ID of the container
+func (c *Container) ID() string {
+	return c.config.ID
+}
+
+// GetArtifact returns a container's artifacts
+func (c *Container) GetArtifact(name string) ([]byte, error) {
+	var data []byte
+	reply, err := iopodman.ContainerArtifacts().Call(c.Runtime.Conn, c.ID(), name)
+	if err != nil {
+		return nil, err
+	}
+	if err := json.Unmarshal([]byte(reply), &data); err != nil {
+		return nil, err
+	}
+	return data, err
+}
+
+func (c *Container) inspect(size bool) (*inspect.ContainerInspectData, error) {
```

Comment:
Why duplicate the public method "Inspect()"?

Reply:
purley by mistake I think. removed